Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -389,6 +389,9 @@ private void SendException(string operationName, string problemId, ExceptionTele
                     exceptionTelemetry.ProblemId = problemId;
                 }
 
+                // this property allows to differenciate examples from regular exceptions tracked using TrackException
+                exceptionTelemetry.Properties.Add("_MS.ProcessedByMetricExtractors", "(Name: Exceptions, Ver: 1.0)");
```

Comment:
What's the reason behind this format, does this format simplify parsing on UI?
Is "_MS.ProcessedByMetricExtractors" used somewhere else as well, so it's a standard name? It adds ~20 bytes to each metrics due to "ByMetricExtractor", we can shorten it if not standard.